src/temp/ProjectImagesSlider/Slider.module.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../theme/theme';
+
+.imageSliderContainer {
+    max-width: 306px;
+}
+
+.sliderContent {
+    width: 100%;
+    height: 100%;
+    border-radius: 13px;
+    background-position: center center !important;
+    background-size: cover !important;
+}
+.sliderContentText {
+    position: absolute;
+    bottom: 38px;
+    left: 18px;
+    color: $backgroundColor;
+    font-size: $fontXSmall;
+    text-transform: capitalize;
+    max-width: 250px;
```

Comment:
The text and progress indicator is not properly responsive. Perhaps provide `max-width: 90%` so that the available space for this adjusts with the larger container.